ethicalads-theme/templates/ea/advertisers-hiring.html

```diff
@@ -84,61 +84,90 @@ <h3>Find your niche</h3>
         <h3>We know developers</h3>
 
         <!-- Text -->
-        <p class="text-gray-800 mb-0">Developers are hard to reach but that's where we can help. Every site on our publisher network is hand-approved for both topic relevance and quality.</p>
+        <p class="text-gray-800 mb-0">Reach developers effortlessly on our 100% hand-approved, exclusively developer-centric network.</p>
 
       </div>
     </div> <!-- / .row -->
   </div> <!-- / .container -->
 </section>
 
 
+<!-- Quote & CTA -->
+<!-- ================================================== -->
+<section class="position-relative mt-5 mb-10">
+
+  <!-- Content -->
+  <div class="container">
+
+    <!-- Heading -->
+    <div class="text-center">
+      <h2 class="text-center font-weight-bold mb-8">Trusted by the best</h2>
+    </div>
+
+    <div class="row mb-10">
+
+      <div class="col-12 col-md-6">
+
+        <div class="w-md-75 mx-auto">
+          <!-- Heading -->
+          <h1 class="mt-10 display-3">
```

Comment:
```suggestion
          <h1 class="mt-5 display-3">
```

Fixes a small alignment issue.